htdocs/admin/stock.php

```diff
@@ -660,13 +659,17 @@
 print '<td class="right">'.$langs->trans("Status").'</td>'."\n";
 print '</tr>'."\n";
 
+print '<form method="POST" action="'.$_SERVER['PHP_SELF'].'">';
```

Comment:
Using 'form' tag inside closing and opening tr is no more allowed. This create javascript errors with some js libraries.